vspec/model/constants.py

```diff
@@ -59,46 +83,79 @@ class StringStyle(Enum, metaclass=EnumMetaWithReverseLookup):
-    MILIMETER = "mm"
-    CENTIMETER = "cm"
-    METER = "m"
-    KILOMETER = "km"
-    KILOMETERPERHOUR = "km/h"
-    METERSPERSECONDSQUARED = "m/s^2"
-    LITER = "l"
-    DEGREECELCIUS = "celsius"
-    DEGREE = "degrees"
-    DEGREEPERSECOND = "degrees/s"
-    KILOWATT = "kW"
-    KILOWATTHOURS = "kWh"
-    KILOGRAMM = "kg"
-    VOLT = "V"
-    AMPERE = "A"
-    SECOND = "s"
-    MINUTE = "min"
-    WEEKS = "weeks"
-    MONTHS = "months"
-    UNIXTIMESTAMP = "UNIX Timestamp"
-    PASCAL = "Pa"
-    KILOPASCAL = "kPa"
-    PERCENT = "percent"
-    CUBICMETER = "cm^3"
-    HORSEPOWER = "PS"
-    STARS = "stars"
-    GRAMMPERSECOND = "g/s"
-    GRAMMPERKM = "g/km"
-    KILOWATTHOURSPER100KM = "kWh/100km"
-    LITERPER100KM = "l/100km"
-    LITERPERHOUR = "l/h"
-    MILESPERGALLON = "mpg"
-    POUND = "lbs"
-    NEWTONMETER = "Nm"
-    REVOLUTIONSPERMINUTE = "rpm"
-    INCH = "inch"
-    RATIO = "ratio"
+    """Data Unit Types
+
+    A signal can optionally specify a unit of measurement from the
+    following set. This list composed with definition according to
+    International Units (SI) and few automotive specific units:
+    [Specification](https://www.iso.org/standard/30669.html),
+    [Wikipedia](https://en.wikipedia.org/wiki/International_System_of_Units)
+    """
+    MILIMETER = VSSConstant("mm", "Millimeter", "Distance")
+    CENTIMETER = VSSConstant("cm", "Centimeter", "Distance")
+    METER = VSSConstant("m", "Meter", "Distance")
+    KILOMETER = VSSConstant("km", "Kilometer", "Distance")
+    KILOMETERPERHOUR = VSSConstant("km/h", "Kilometers per hour", "Speed")
+    METERSPERSECONDSQUARED = VSSConstant(
+        "m/s^2",
+        "Acceleration in meters per second squared",
+        "Acceleration"
+    )
+    LITER = VSSConstant("l", "Liter", "Volume")
+    DEGREECELCIUS = VSSConstant("celsius", "Degrees Celsius", "Temperature")
+    DEGREE = VSSConstant("degrees", "Angle in degrees", "Angle")
+    DEGREEPERSECOND = VSSConstant(
+        "degrees/s",
+        "Angular speed in degrees/s",
+        "Angular Speed"
+    )
+    KILOWATT = VSSConstant("kW", "Kilowatt", "Power")
+    KILOWATTHOURS = VSSConstant("kWh", "Kilowatt hours", "Power")
+    KILOGRAMM = VSSConstant("kg", "Kilograms", "Weight")
+    VOLT = VSSConstant("V", "Potential difference in volt", "Electrical")
+    AMPERE = VSSConstant("A", "Current in amperes", "Electrical")
+    SECOND = VSSConstant("s", "Seconds", "Time")
+    MINUTE = VSSConstant("min", "Minutes", "Time")
+    WEEKS = VSSConstant("weeks", "Weeks", "Time")
+    MONTHS = VSSConstant("months", "Months", "Time")
+    UNIXTIMESTAMP = VSSConstant(
+        "UNIX Timestamp",
+        "Seconds since January 1st 1970 UTC",
+        "Time"
+    )
+    PASCAL = VSSConstant("Pa", "Pascal",  "Pressure")
+    KILOPASCAL = VSSConstant("kPa", "Kilo-Pascal", "Pressure")
+    PERCENT = VSSConstant("percent", "Percent", "Relation")
+    CUBICMETER = VSSConstant("cm^3", "Cubic Meters", "Volume")
```

Comment:
Cubic centimeters

Reply:
but which one is wrong (which one should I change)? The value or the name, see https://github.com/GENIVI/vss-tools/blob/master/vspec/model/constants.py#L85 it's already `CUBICMETER = "cm^3"` 👀

Reply:
Our only use of `cm^3` in VSS is for displacement where we in the description clearly says cubic centimetres (and that is the unit that makes sense, if you are to represent it as a uint16. So I would say changing to "CUBICCENTIMETER" makes sense. I have a vague memory that we spotted this error earlier at some time, but it seems we did not fix it.

(And it seems that we prefer US spelling generally, e.g. centimeter and tire)

```
Displacement:
  datatype: uint16
  type: attribute
  description: Displacement in cubic centimetres.
  unit: cm^3
```

Reply:
fixed in the updated branch, I added this as a first commit, independent from the VSSConstants and scripts so it can be merged regardless of the tools. I won't open another PR yet, but if this PR is rejected then I can create another one with just thix fix